extensions/amp-base-carousel/1.0/test/test-base-carousel.js

```diff
@@ -93,6 +93,30 @@ describes.sandboxed('BaseCarousel preact component', {}, () => {
     expect(slides.at(2).text()).to.equal('slide 2');
   });
 
+  it('should snap to slides by default', () => {
+    const jsx = (
```

Comment:
Nit, but we should definitely be inlining jsx inside the `mount()` call.

Reply:
Done.